src/libsync/discoveryphase.cpp

```diff
@@ -403,9 +401,8 @@ void DiscoverySingleDirectoryJob::directoryListingIteratedSlot(QString file, con
         }
         if (_isExternalStorage) {
             /* All the entries in a external storage have 'M' in their permission. However, for all
-               purposes in the desktop client, we only need to know about the mount points.
-               So replace the 'M' by a 'm' for every sub entries in an external storage */
-            file_stat->remotePerm.replace('M', 'm');
+               purposes in the desktop client, we only need to know about the mount points. */
+            file_stat->remotePerm.setPermission(RemotePermissions::IsMountedSub);
```

Comment:
This means you will now write `'M'` + `'m'` in the database, is this better making them mutually exclusive?

Reply:
then i need to have a unsetPermission

Reply:
It doesn't seem to cause a problem now, but if we don't update old 'm' to also have 'M', we'll have both styles in databases now, potentially creating confusion down the line.